lib/sneakers/configuration.rb

```diff
@@ -79,6 +79,7 @@ def merge!(hash)
           hash.delete(k)
           @hash.delete(k)
         end
+        hash.merge!(share_bunny_connection: true) # Share the connection when set.
```

Comment:
This forces `share_bunny_connection`. Is the intent to simply change the default?